common/src/model_registration/register.py

```diff
@@ -166,9 +174,16 @@ def main(args):
 
     if model_type == "mlflow_model":
         # Make sure parent directory is mlflow_model_folder for mlflow model
-        shutil.copytree(model_path, "mlflow_model_folder", dirs_exist_ok=True)
-        model_path = "mlflow_model_folder"
+        print("Model is of type MLFlow.")
+
+        target_dir = os.path.join(model_path, "mlflow_model_folder")
```

Comment:
Can we define this as some constant and then use it, instead of hardcoding here

Reply:
Done